pkg/alexa/model.go

```diff
@@ -8,8 +8,8 @@ type Model struct {
 // InteractionModel defines the base model structure
 type InteractionModel struct {
 	Language LanguageModel  `json:"languageModel"`
-	Dialog   *Dialog        `json:"dialog"`
-	Prompts  *[]ModelPrompt `json:"prompts"`
+	Dialog   *Dialog        `json:"dialog,omitempty"`
+	Prompts  *[]ModelPrompt `json:"prompts,omitempty"`
```

Comment:
Nope. Do not have a pointer to an slice.

Reply:
How do I omit the element in JSON output then? I don't want `"foobar": []`, I just want it to not appear...

Reply:
You wont get that, cause of the `omitempty`. It knows when the slice is empty, it doesnt with objects, hence the ptr on objects.

Reply:
fixed.